Misc/NEWS.d/next/Library/2019-06-03-23-53-25.bpo-27513.qITN7d.rst

```diff
@@ -0,0 +1,2 @@
+:func:`email.charset.getaddresses` can now handle
```

Comment:
The import path here is wrong, you meant, `email.utils.getaddress`. Also a stray `~` before the class import path.

Also, I suggest a slight reword:

```
:func:`email.utils.getaddress` now accepts :class:`email.header.Header` objects along with string values. Patch by Zackery Spytz.
```
What do you think?